src/Passwordless/Models/Credential.cs

```diff
@@ -1,7 +1,28 @@
 ﻿namespace Passwordless;
 
+/// <summary>
+/// Store credential used to authenticate users. Follows WebAuthN specification
+/// <see href="https://www.w3.org/TR/webauthn-2/#public-key-credential"/>
+/// </summary>
 public class Credential
 {
+    /// <summary>
+    /// Initializes an instance of <see cref="Credential"/>
+    /// </summary>
+    /// <param name="descriptor"><see cref="Descriptor"/></param>
+    /// <param name="publicKey"><see cref="PublicKey"/></param>
+    /// <param name="userHandle"><see cref="UserHandle"/></param>
+    /// <param name="signatureCounter"><see cref="SignatureCounter"/></param>
+    /// <param name="attestationFmt"><see cref="AttestationFmt"/></param>
+    /// <param name="createdAt"><see cref="CreatedAt"/></param>
+    /// <param name="aaGuid"><see cref="AaGuid"/></param>
+    /// <param name="lastUsedAt"><see cref="LastUsedAt"/></param>
+    /// <param name="rpid"><see cref="RPID"/></param>
+    /// <param name="origin"><see cref="Origin"/></param>
+    /// <param name="country"><see cref="Country"/></param>
+    /// <param name="device"><see cref="Device"/></param>
+    /// <param name="nickname"><see cref="Nickname"/></param>
+    /// <param name="userId"><see cref="UserId"/></param>
```

Comment:
We could also just not include the params in the docs if it makes things messy for no significant value